troposphere/static/js/components/modals/nullProject/NullProjectMigrateResourceModal.react.js

```diff
@@ -169,7 +168,12 @@ define(
       },
 
       renderBody: function(){
-        if(this.state.projects) {
+        if(this.state.projects == null) {
```

Comment:
This was the sort of lines i was referring to, where .length == 0 would evaluate to 'falsey' here but having an empty collection should evaluate slightly differently..